est/001.est

```diff
@@ -0,0 +1,22 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: Design the operation commands
```

Comment:
@dalifreire What do you mean here exactly in terms of coding, implementing the commmands?
If so how is this different from 3. ?